logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -104,8 +104,12 @@ interface ConversationRepository {
 
     suspend fun getConversationList(): Either<StorageFailure, Flow<List<Conversation>>>
     suspend fun observeConversationList(): Flow<List<Conversation>>
-    suspend fun getProteusTeamConversations(teamId: TeamId): Either<StorageFailure, List<QualifiedID>>
-    suspend fun getProteusTeamConversationsReadyForFinalisation(teamId: TeamId): Either<StorageFailure, List<QualifiedID>>
+    suspend fun getConversationIds(
+        type: Conversation.Type,
+        protocol: Conversation.Protocol,
+        teamId: TeamId? = null
+    ): Either<StorageFailure, List<QualifiedID>>
+    suspend fun getTeamConversationIdsReadyForFinalisation(teamId: TeamId): Either<StorageFailure, List<QualifiedID>>
```

Comment:
hmm, it's good but what finalization? :D

Reply:
So you mean 
```suggestion
    suspend fun getTeamConversationIdsReadyForMigrationFinalisation(teamId: TeamId): Either<StorageFailure, List<QualifiedID>>
```

Reply:
Or maybe

```suggestion
    suspend fun getTeamConversationIdsReadyToCompleteMigration(teamId: TeamId): Either<StorageFailure, List<QualifiedID>>
```

Reply:
The last one seems far better thanks